Source/WebCore/rendering/RenderReplaced.cpp

```diff
@@ -275,6 +275,11 @@ void RenderReplaced::paint(PaintInfo& paintInfo, const LayoutPoint& paintOffset)
         return;
     }
 
+    if (paintInfo.phase == PaintPhase::ClippingMask && style().usedVisibility() == Visibility::Visible) {
+        paintClippingMask(paintInfo, adjustedPaintOffset);
+        return;
+    }
+
```

Comment:
No related. The previous statement does not have the check `style().usedVisibility() == Visibility::Visible`. Is this intentionally dropped? Or is it mistakenly missing?
```
    if (paintInfo.phase == PaintPhase::Mask) {
        paintMask(paintInfo, adjustedPaintOffset);
        return;
    }
```

Reply:
Oh I see what is happening. `RenderReplaced::shouldPaint()` have this statement:
```
    // if we're invisible or haven't received a layout yet, then just bail.
    if (style().usedVisibility() != Visibility::Visible)
        return false;
```
So maybe we do not need to check `style().usedVisibility() == Visibility::Visible` again in `RenderReplaced::paint()`.